src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/VisualRuleForm.tsx

```diff
@@ -178,33 +165,74 @@ export const VisualRuleForm: React.FC<VisualRuleFormProps> = (props) => {
                             <Dropdown
                                 required
                                 placeholder={t(
-                                    '3dSceneBuilder.visualRule.typePlaceholder'
+                                    '3dSceneBuilder.visualRuleForm.typePlaceholder'
                                 )}
                                 label={t('type')}
                                 selectedKey={`value-type-${formData.propertyType}`}
                                 onChange={onTypeChange}
                                 options={typeOptions}
                                 onRenderOption={onRenderTypeOption}
                                 onRenderTitle={onRenderTypeTitle}
+                                styles={classNames.subComponentStyles.dropdown}
                             />
                         )}
                     </Stack>
                 </div>
                 <Separator />
-                <div className={commonFormStyles.expandedSection}>
-                    <ConditionsList />
+                <div className={commonFormStyles.expandingSection}>
+                    <Stack horizontal verticalAlign={'center'}>
+                        <Label styles={classNames.subComponentStyles.label}>
+                            {t('3dSceneBuilder.conditions')}
+                        </Label>
+                        <TooltipCallout
+                            content={{
+                                buttonAriaLabel: t(
+                                    '3dSceneBuilder.visualRuleForm.conditionsInfoContent'
+                                ),
+                                calloutContent: t(
+                                    '3dSceneBuilder.visualRuleForm.conditionsInfoContent'
+                                )
+                            }}
+                            styles={
+                                classNames.subComponentStyles.tooltipCallout
+                            }
+                        />
+                    </Stack>
+                    <ConditionsList
+                        styles={classNames.subComponentStyles.conditionsList}
+                    />
                 </div>
             </div>
             <PanelFooter>
                 <PrimaryButton
                     text={t('save')}
-                    onClick={() => setVisualRuleFormMode(null)}
+                    onClick={() =>
+                        setVisualRuleFormMode(VisualRuleFormMode.Inactive)
+                    }
+                    styles={
+                        classNames.subComponentStyles.saveButton
```

Comment:
nit: you might know this already but the shorthand for this is `classNames.subComponentStyles.saveButton?.()`